packages/dashboard-v2/src/components/CurrentPlan/CurrentPlan.js

```diff
@@ -28,17 +29,20 @@ const CurrentPlan = () => {
   }
 
   return (
-    <div>
+    <div className="flex flex-col h-full">
       <h4>{activePlan.name}</h4>
-      <div className="text-palette-400">
-        {activePlan.price === 0 && <p>100GB without paying a dime! 🎉</p>}
+      <div className="text-palette-400 justify-between flex flex-col grow">
+        {activePlan.price === 0 && activePlan.limits && (
+          <p>{prettyBytes(activePlan.limits.storageLimit)} without paying a dime! 🎉</p>
```

Comment:
I think you need to pass binary flag here (see https://account.siasky.net/api/limits)

```suggestion
          <p>{prettyBytes(activePlan.limits.storageLimit, { binary: true })} without paying a dime! 🎉</p>
```

Reply:
a392863c